src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Models/LocalConfigurationClient.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Azure.Data.AppConfiguration;
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Models
```

Comment:
I don't understand where the `Models` namespace comes from

Reply:
Ah I see, this namespace already exists. Well I don't believe that a `Client` constitutes a model in the generic sense.